frontend/src/components/AuthValidationErrors.tsx

```diff
@@ -0,0 +1,50 @@
+import { TiInfo } from 'react-icons/ti'
+import { IoClose } from 'react-icons/io5'
+import { classNames } from '~/utils/classNames'
+import React from 'react'
+
```

Comment:
separate the imports between our own imports and library imports
```suggestion
import { TiInfo } from 'react-icons/ti'
import { IoClose } from 'react-icons/io5'
import React from 'react'

import { classNames } from '~/utils/classNames'

```